xml/depl_maintenance.xml

```diff
@@ -312,6 +312,175 @@
   </orderedlist>
  </sect1>
 
+ <sect1 xml:id="sec.depl.maintenance.recover.compute.node.failure">
+  <title>Recovering from &compnode; Failure</title>
+  <para>
+   The following procedure assumes that there is at least one &compnode;
+   already running. Otherwise, see <xref
+   linkend="sec.depl.maintenance.bootstrap.compute.plane" />.
+  </para>
+  <procedure>
+   <step>
+    <para>
+     If the &compnode; failed, it should have been fenced. Verify that this is
+     the case. Otherwise, check <filename>/var/log/pacemaker.log</filename> on
+     the Designated Co-ordinator to determine why the &compnode; was not fenced. The
+     most likely reason is a problem with STONITH devices.
+    </para>
+   </step>
+   <step>
+    <para>
+     Determine the cause of the &compnode;'s failure.
+    </para>
+   </step>
+   <step>
+    <para>
+     Boot the &compnode; again.
+    </para>
+   </step>
+   <step>
+    <para>
+     Check whether the <systemitem>crowbar_join</systemitem> script ran
+     successfully on the &compnode;. If this is not the case, check the log
+     files to find out the reason.
+    </para>
+   </step>
+   <step>
+    <para>
+     If the <systemitem>chef-client</systemitem> agent triggered by
+     <systemitem>crowbar_join</systemitem> succeeded, confirm that the
+     <systemitem>pacemaker_remote</systemitem> service is up and
+     running.
+    </para>
+   </step>
+   <step>
+    <para>
+     Check whether the remote node is registered and considered healthy by the
+     core cluster. If this is not the case check
+     <filename>/var/log/pacemaker.log</filename> on the Designated Co-ordinator
```

Comment:
Our SLEHA docs use "Designated Coordinator" (no hyphen) which also makes more sense to me.